typescript/infra/config/environments/testnet4/chains.ts

```diff
@@ -16,4 +16,9 @@ export const chainMetadataOverrides: ChainMap<Partial<ChainMetadata>> = {
       gasPrice: 8 * 10 ** 9, // 8 gwei
     },
   },
+  scrollsepolia: {
+    transactionOverrides: {
+      gasPrice: 5 * 10 ** 8, // 0.5 gwei
```

Comment:
curious if this was bc of issues you saw or just preventative given we do something similar on mainnet

Reply:
I added it because of issues I saw - without it we would reliably fail to get any txs included